src/DependencyInjection/HostnetEntityTrackerExtension.php

```diff
@@ -34,7 +34,10 @@ public function load(array $configs, ContainerBuilder $container)
         if (array_key_exists('blamable', $config)) {
             $this->validateComponent(self::BLAMABLE, 'blamable');
             $loader->load('blamable.yml');
-            if (class_exists('Symfony\Bundle\SecurityBundle\SecurityBundle')) {
+            if (
+                $container->hasParameter('kernel.bundles')
+                && key_exists('SecurityBundle', $container->getParameter('kernel.bundles'))
```

Comment:
use array_key_exists instead of key_exists.

key_exists — Alias of array_key_exists() -> http://php.net/manual/en/function.key-exists.php